docs/build.rst

```diff
@@ -192,19 +192,42 @@ HDF5 features in h5py itself::
     $ pip install --no-binary=h5py h5py
 
 If you want access to the full Parallel HDF5 feature set in h5py
-(:ref:`parallel`), you will further have to build in MPI mode.  This can either
-be done with command-line options from the h5py tarball or by::
+(:ref:`parallel`), you will further have to build in MPI mode.
 
-    $ export HDF5_MPI="ON"
```

Comment:
why is this export no longer needed?

Reply:
I thought that by using --mpi in the configure step, the environment variable export was superfluous?